packages/cloud_firestore/lib/src/query.dart

```diff
@@ -162,6 +170,46 @@ class Query {
     return _copyWithParameters(<String, dynamic>{'orderBy': orders});
   }
 
+  /// Takes a documentSnapshot, creates and returns a new [Query] that starts after
+  /// the provided document reference to the order of the query.
+  ///
+  /// documentSnapshot must be in order of [orderBy] filters.
```

Comment:
document sentence need to start with capital letters. also highlight the parameters Maybe: 
```suggestion
  /// The `documentSnapshot` must be in order of [orderBy] filters.
```
Same applies to other new query methods

Reply:
This sentence actually just doesn't make sense, removing